addons/phone_validation/tools/phone_validation.py

```diff
@@ -18,15 +19,15 @@ def phone_parse(number, country_code):
             phone_nbr = phonenumbers.parse(number, region=country_code or None, keep_raw_input=True)
         except phonenumbers.phonenumberutil.NumberParseException as e:
             raise UserError(
-                _('Unable to parse %(phone)s: %(error)s', phone=number, error=str(e))
+                _lt('Unable to parse %(phone)s: %(error)s', phone=number, error=str(e))
```

Comment:
Try to pass some `env` in parameters, if possible.

Reply:
Too many usages.